.travis.yml

```diff
@@ -32,6 +29,12 @@ env:
 
 matrix:
   include:
+    - env: TOXENV=py26
```

Comment:
These changes are required with the migration of default Travis images from Precise to Trusty.